### PR TITLE
Tweak FP handling FAQ

### PR DESCRIPTION
**Pull request by kingthorin**
- Tweak report statement to be more up-to-date. 
- Add API tip.

### CODE REVIEW

**Comment by kingthorin** [comment]
Submitted via web UI, settings DCO pass (per all my other recent PRs).

File: site/content/faq/how-do-i-handle-a-false-positive.md [hunk 1]
```diff
@@ -18,4 +18,11 @@ If you are automating ZAP and the relevant scan rule does not support a suitable
 
 The [packaged scans](/docs/docker/) also support regex expressions in the [configuration file](/docs/docker/api-scan/#configuration-file) which allow you to ignore alerts for the URLs you specify.
 
-The built in report does not include false positives, but some add-ons which provide reporting capabilities may still do.
+The reports does not include false positives by default, however you can choose to filter the included alerts by Risk or Confidence.
+
+If you wish to adjust risk or confidence via the API check the following endpoints:
+```
+updateAlert (id* name* riskId* confidenceId* description* param attack otherInfo solution references evidence cweId wascId )
```
Comment (psiinon) [inline site/content/faq/how-do-i-handle-a-false-positive.md:25]: These could link to the API docs?

Comment (kingthorin) [reply site/content/faq/how-do-i-handle-a-false-positive.md:25]: Sadly they're newer than the API docs, and updating them is currently problematic.

**Comment by kingthorin** [comment]
Fixed

**Review by psiinon — APPROVED** [review]

File: site/content/faq/how-do-i-handle-a-false-positive.md (hunk 1 above)
Comment (thc202) [reply site/content/faq/how-do-i-handle-a-false-positive.md:25]: Worth being specific where they are e.g. `/alert/updateAlert…`

Comment (thc202) [reply site/content/faq/how-do-i-handle-a-false-positive.md:25]: Or mention the component in the previous sentence.

Comment (kingthorin) [reply site/content/faq/how-do-i-handle-a-false-positive.md:25]: Okay will tweak that

**Comment by kingthorin** [comment]
Tweaked API endpoints' path.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**